docs/getting-started/index.md

```diff
@@ -69,9 +71,20 @@ WSL may be having networking issues.  To resolve this, you can follow the steps
 ### Set Up a New Test Kit
 {:toc-skip}
 
-1. Clone the [Inferno Template](https://github.com/inferno-framework/inferno-template) repository. You can
-   either clone this repository directly, or click the green "Use this template"
-   button to create your own repository based on this one.
+There are two convenient methods to create a new Test Kit. The first is by cloning the 
+[Inferno Template](https://github.com/inferno-framework/inferno-template) repository. This
+method ensures you will always be starting with the latest Test Kit Template. You can
+either clone this repository directly, or click the green "Use this template" button in GitHub
+to create your own repository based on the Inferno Template.
+
+The second method is to use the Inferno Command Line Interface. After installing
+the `inferno_core` gem, run `inferno new TestKitName` to create a new Test Kit
+Template in a directory named `test-kit-name`. See how to [create a new Test
+Kit](/docs/getting-started/inferno-cli.html#creating-a-new-test-kit) using the Inferno CLI for more
+information and options.
+
```

Comment:
I'd like copy-pasteable code blocks:
```
gem install inferno_core && inferno new my-test-kit
```

Reply:
I like copy buttons on code blocks too, but I think I'll open a separate ticket for that functionality.